modules/data.land/R/extract_FIA.R

```diff
@@ -11,9 +11,12 @@
 ##' @title extract_FIA
 ##' @export
 ##' @author Istem Fer
-extract_FIA <- function(lon, lat, start_date, end_date, gridres = 0.075, dbparms, ...){
```

Comment:
Can you explain why this rename was necessary? Spelling these arguments as `start_date` and `end_date` is standard enough across enough different PEcAn modules that I strongly encourage keeping the underscores here too if it's possible.

Reply:
Yes I will change it back, this is a relic of the debugging phase. I do not know the exact reason but when I was debugging the ic_process workflow I would get weird errors sometimes from passing start_date that disappeared when I changed it to startdate.